osmaxx/excerptexport/static/excerptexport/scripts/maps/draw_controls.js

```diff
@@ -96,9 +138,13 @@ var draw_controls = function (map) {
             var estimatedFileSize = Number(data['estimated_file_size_in_bytes']);
 
             var message_html = '';
-            if (estimatedFileSize > allowedMaxSize) {
-                var howMuchTooLarge = estimatedFileSize ? Math.ceil(estimatedFileSize * 100 / allowedMaxSize - 100) + '% ' : '';
+            if (isExtentTooLarge(estimatedFileSize)) {
+                var howMuchTooLarge = estimatedFileSize ? Math.ceil(estimatedFileSize * 100 / getAllowedMaxSize() - 100) + '% ' : '';
                 var message = 'Excerpt {percent}too large!'.replace('{percent}', howMuchTooLarge);
+
+                if (!isSimplifiedChosen()) {
+                    message += ' Switching to the simplified data-model allows larger extends.';
```

Comment:
Should this message be displayed when the simplified export would be too large, too?